ionic/__main__.py

```diff
@@ -1,15 +1,19 @@
 import argparse
 import asyncio
 import datetime as dt
+from datetime import datetime, timedelta
```

Comment:
Could we switch to using dt.datetime and dt.timedelta everywhere since the code already imports datetime as dt just above it?

Reply:
Absolutely, python namespaces had me confused. I will remove the line and swap to dt.datetime and dt.timedelta

Reply:
Much appreciated!